### ISSUE TITLE
Massive amounts of lua errors (7.2.5)

### ISSUE THREAD

**Issue by Deharme** [cogwerkz/gUI4#12]
**So far error occurs only in dungeons when NPC begins dialogue or any of the players types something in chat. Errors started poping after latest patch 7.2.5.**

Message: Interface\AddOns\gUI4_Chat\elements\bubbles.lua:79: Attempt to access forbidden object from code tainted by an AddOn
Time: 06/16/17 19:15:34
Count: 21329
Stack: Interface\AddOns\gUI4_Chat\elements\bubbles.lua:79: Attempt to access forbidden object from code tainted by an AddOnInterface\SharedXML\SharedBasicControls.lua:204: in function <Interface\SharedXML\SharedBasicControls.lua:203>
[C]: ?
[C]: in function `GetName'
Interface\AddOns\gUI4_Chat\elements\bubbles.lua:79: in function `IsBubble'
Interface\AddOns\gUI4_Chat\elements\bubbles.lua:94: in function <Interface\AddOns\gUI4_Chat\elements\bubbles.lua:89>

Locals: errorMessage = "Interface\AddOns\gUI4_Chat\elements\bubbles.lua:79: Attempt to access forbidden object from code tainted by an AddOn"
DisplayMessageInternal = <function> defined @Interface\SharedXML\SharedBasicControls.lua:187
MESSAGE_TYPE_ERROR = 0

**Comment by Landelor** [comment]
Chat bubbles in Dungeons are now a protected item.
I would disable GUI chat bubbles for now. Project has seen much movement of late.

**Comment by goldpaw** [comment]
Appears you're right. This is technically the same bug we experienced in 7.2.0 when friendly NPC nameplates in instances became protected in this manner. I solved that, but now that it appears that Blizzard have protected instance chat bubbles in the same manner, I guess I really have to disable the bubbles while inside instances. 

Can't say I'm loving this, as big instance bubble spam was the reason I wrote this module in the first place. But it is how it is, sometimes blizzard make changes we simply can't work around.
